### PR TITLE
Making help messages about PRs plural

### PR DESCRIPTION
**Pull request by bradmwilliams**

### CODE REVIEW

File: pkg/slack/slack.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func NewBot(botToken, botSigningSecret string, graceperiod time.Duration, port i
 
 func (b *Bot) SupportedCommands() []parser.BotCommand {
 	return []parser.BotCommand{
-		parser.NewBotCommand("launch <image_or_version_or_pr> <options>", &parser.CommandDefinition{
+		parser.NewBotCommand("launch <image_or_version_or_prs> <options>", &parser.CommandDefinition{
 			Description: fmt.Sprintf("Launch an OpenShift cluster using a known image, version, or PR. You may omit both arguments. Use `nightly` for the latest OCP build, `ci` for the the latest CI build, provide a version directly from any listed on https://amd64.ocp.releases.ci.openshift.org, a stream name (4.1.0-0.ci, 4.1.0-0.nightly, etc), a major/minor `X.Y` to load the \"next stable\" version, from nightly, for that version (`4.1`), `<org>/<repo>#<pr>` to launch from a PR, or an image for the first argument. Options is a comma-delimited list of variations including platform (%s) and variant (%s).",
 				strings.Join(CodeSlice(manager.SupportedPlatforms), ", "),
 				strings.Join(CodeSlice(manager.SupportedParameters), ", ")),
```
Comment (bparees) [inline pkg/slack/slack.go:86]: there's an example one line below this of referencing a PR.  If you changed the example to illustrate referencing multiple PRs, i think that would be helpful (would illustrate the comma separated syntax).

similarly anywhere else we have examples that reference a PR.  I think it's easier for people to generalize an example of multiple items down to how to do it for one item, than the inverse.

**Comment by bradmwilliams** [comment]
/hold
Working on a few more changes

**Comment by bradmwilliams** [comment]
I've updated the help messaging and formatting a bit and added a link to the FAQ.  Hopefully, this will be a little easier to read and digest for folks.
help

File: pkg/slack/slack.go (hunk 1 above)
Comment (bradmwilliams) [reply pkg/slack/slack.go:87]: Added example to all commands that take options.

**Pull request merged**